packages/neuron-wallet/src/models/subjects/address-created-subject.ts

```diff
@@ -1,14 +1,16 @@
 import { ReplaySubject } from 'rxjs'
 import { Address } from 'database/address/address-dao'
+import { remote } from 'electron'
+import ProcessUtils from 'utils/process'
 
 export default class AddressCreatedSubject {
```

Comment:
If is it possible to combine this subject into the address DB changed subject.

Reply:
Seems no need, different info / sender / listener